app/templates/components/copyright-item.hbs

```diff
@@ -0,0 +1,9 @@
+<div class='copyright element'>
```

Comment:
This can be removed and instead the class can be added in the component definition.

See https://guides.emberjs.com/v2.6.0/components/customizing-a-components-element/